Shortest Path algorithm

```diff
@@ -0,0 +1,44 @@
+#include<iostream>
```

Comment:
Incorrect file extension and add this to a folder.
Name of algorithm is required.